src/pages/all-about-carbon/the-team.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: The team
+description:
+  Carbon is not just one team. To sustain productive contribution and maintain a
+  clear direction for Carbon as a whole, various groups works together to
+  provide direction, governance, and support.
+---
+
+<PageDescription>
+
+Carbon is not just one team. To sustain productive contribution and maintain a
+clear direction for Carbon as a whole, various groups works together to provide
+direction, governance, and support.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Overview</AnchorLink>
+  <AnchorLink>The Carbon team</AnchorLink>
+  <AnchorLink>Steering committee</AnchorLink>
+  <AnchorLink>Working groups</AnchorLink>
+  <AnchorLink>Board of directors</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Carbon is funded and built by IBM. That means we build for the company’s
+business needs. Carbon is also an open source community-driven project. It is
+available for anyone to use and everyone is encouraged to contribute.
+
+To sustain productive contribution and maintain a clear direction for Carbon as
+a whole, there are various groups within IBM that provide governance and
+support.
+
+## The Carbon team
+
+Carbon is maintained by a dedicated team of designers, developers, and writers
+based in the Austin, Texas IBM Design Studio.
+
+## Steering committee
+
+The Carbon Design System Steering Committee provides oversight and direction for
+Carbon and its design and development community, and related projects.
+
+### Responsibilities
+
+The key responsibilities of the committee are as follows:
+
+- Improve design strategy, quality, and consistency within the design system.
+- Promote and support usage of and contribution to the Carbon Design System
+  within their organizations.
+- Identify and align components/patterns/content that are multi-product or
+  universal.
+- Facilitate consolidation of components and patterns where multiples exist -
+  i.e. “pick winners”.
+- Assess and vote to approve or deny community proposals (e.g. if a particular
```

Comment:
Instead of "community proposals", I like the word "projects". E.g. Carbon Angular is a project. Carbon for Security is a project. Carbon Mobile will be a project. We could re-write this as:

> Assess and approve Carbon projects (e.g. any repository, asset, website, etc. that bears the Carbon name.)

Reply:
On that same note, let's consolidate "projects" and "working groups".

Reply:
I agree. Let's conflate "community proposals" into "projects". So actually remove the "community proposal" bit, and just focus on "projects" as a replacement for "working groups".